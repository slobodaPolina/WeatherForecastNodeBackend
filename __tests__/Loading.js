import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../src/components/Loading';

it('renders loading correctly', () => {
  const tree = renderer.create(<Loading/>).toJSON();
  expect(tree).toMatchSnapshot();
});
