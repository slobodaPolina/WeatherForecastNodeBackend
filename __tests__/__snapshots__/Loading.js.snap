// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loading correctly 1`] = `
<div>
  <div
    className="bigText"
  >
    Подождите, данные загружаются
  </div>
  <div
    className="refreshBanner"
  >
    <img
      alt="loading"
      className="refreshImage"
      src="/images/refresh.svg"
    />
  </div>
</div>
`;
